index.html

```diff
@@ -99,22 +108,29 @@ <h2>There is a flavor of infrastructure for everyone on Crossplane</h2>
           alt="Compose infrastructure for your applications" />
       </div>
       <div class="col-6_md-12">
-        <h2>Publish simplified infrastructure abstractions for your applications</h2>
+        <h2>Putting you in control</h2>
         <p>
-          Build your own internal infrastructure abstractions on top of the CRDs Crossplane provides. Your custom APIs can include policy guardrails, hiding
-           infrastructure complexity and making it safe for applications to consume.
+          Most platforms require that you buy into their opinionated API
+          concepts. With Crossplane you can build a platform around your own
+          opinions. We know the best control planes are tailored to the task at
+          hand so we designed Crossplane as a framework that puts you in
+          control. Use Crossplane to design a control plane that exposes
+          declarative APIs tailored to your unique orchestration needs.
           <img src="{{ "/images/quad-teal-medium.svg" | relative_url }}" />
         </p>
       </div>
     </div>
     <div class="grid">
       <div class="col-6_md-12">
-        <h2>The Universal Cloud API</h2>
+        <h2>Control planes - the future of cloud</h2>
         <p>
-          Crossplane provides a consistent API across a diverse set of vendors, resources, and
-          abstractions. The Crossplane Resource Model (XRM) extends the Kubernetes Resource Model
-          (KRM) in an opinionated way, resulting in a uniform experience for managing resources,
-          no matter what provider or vendor built them.
+          The cloud vendors have been building with control planes for years.
+          Now Crossplane helps you do the same. Control planes are
+          self-healing—they automatically correct drift. Consumers can
```

Comment:
```suggestion
          self-healing — they automatically correct drift. Consumers can
```

Reply:
spaces around this hyphen/separator? it looks weird to not have spaces when it's smooshed together with the hyphen in self-healing 🤔

Reply:
I actually copied this "spaceless" pattern from the existing site. See the "Created to power a more open cloud" section.

Reply:
ah got it - if that's the existing pattern then that is reasonable :)

Reply:
> it looks weird to not have spaces when it's smooshed together with the hyphen in self-healing

It actually renders out as two different characters when its not in fixed width font. The dash in self-healing is a hyphen while the following one is a longer em-dash.